src/scr.c

```diff
@@ -1656,6 +1659,23 @@ int SCR_Init()
   scr_cindex_file = spath_from_str(scr_cntl_prefix);
   spath_append_strf(scr_cindex_file, "cindex.scrinfo", scr_storedesc_cntl->rank);
 
+  /* store parameters set by app code for use by post-run scripts */
+  if (scr_my_rank_world == 0) {
+    spath* path_app_config = spath_from_str(scr_prefix_scr);
+    spath_append_str(path_app_config, SCR_CONFIG_FILE_APP);
+    spath_reduce(path_app_config);
+    char *app_config = spath_strdup(path_app_config);
+    spath_delete(&path_app_config);
+
+    scr_param_app_hash_write_file(app_config);
+
+    free(app_config);
```

Comment:
```suggestion
    scr_free(&app_config);
```

Reply:
Doable. The variable (app_config) goes out of scope a line after this so I did not change (I nottd the same free vs. scr_free potential issue). Will change to make this more obvious since I am apparently not the only one taking note of this.

Reply:
Is there a way in GitHub to jump to the current version of those lines to show my fix?